### PR TITLE
Translating docs/docs/recipes/querying-data.md

### PR DESCRIPTION
**Pull request by rnd-debug**

### CODE REVIEW

**Comment by rnd-debug** [comment]
If the PR is accepted, would it be possible to tag it with `hacktoberfest-accepted`? 😇 They have updated their policy in order to reduce spam PRs ([DigitalOcean link](https://hacktoberfest.digitalocean.com/hacktoberfest-update))

File: docs/docs/recipes/querying-data.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 ---
 
-Gatsby lets you access your data across all sources using a single GraphQL interface.
+Gatsby vous permet d'accéder aux données de toutes vos sources en utilisant une seule interface GraphQL.
 
-## Querying data with a Page Query
+## Récupérer les données via une requête de page (Page Query)
 
-You can use the `graphql` tag to query data in the pages of your Gatsby site. This gives you access to anything included in Gatsby's data layer, such as site metadata, source plugins, images, and more.
+Vous pouvez utiliser l'étiquette `graphql` pour récupérer des données dans les pages de votre site Gatsby. Cela vous permet d'avoir accès à tout ce qui est inclus dans la couche de données Gatsby (par exemple, les métadonnées de votre site, les plugins source, les images et bien plus encore).
```
Comment (dantereve) [inline docs/docs/recipes/querying-data.md]: I'm not convinced by "étiquette `graphql`". Can't we still use "tag" here? I've never heard someone talking about some "étiquettes graphql".

File: docs/docs/recipes/querying-data.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ const NonPageComponent = () => (
       data // highlight-line
     ) => (
       <h1>
-        Querying title from NonPageComponent with StaticQuery:
+        Titre récupéré par le composant non-page via une StaticQuery:
```
Comment (dantereve) [inline docs/docs/recipes/querying-data.md]:
```suggestion
        Titre récupéré par NonPageComponent via une StaticQuery :
```
C'est le nom du composant pas besoin de le traduire ici 😉

File: docs/docs/recipes/querying-data.md [hunk 3]
```diff
@@ -140,28 +140,28 @@ const NonPageComponent = () => {
...
 ```
 
-### Additional resources
+### Ressources additionnelles
 
-- [More on Static Query for querying data in components](/docs/static-query/)
-- [The difference between a static query and a page query](/docs/static-query/#how-staticquery-differs-from-page-query)
-- [More on the useStaticQuery hook](/docs/use-static-query/)
-- [Visualize your data with GraphiQL](/docs/introducing-graphiql/)
+- Plus sur [l'utilisation de StaticQuery dans les composants pour récupérer les données](/docs/static-query/)
```
Comment (dantereve) [inline docs/docs/recipes/querying-data.md]:
```suggestion
- [Plus sur l'utilisation de StaticQuery dans les composants pour récupérer les données](/docs/static-query/)
```

**Review by dantereve — DISMISSED** [review]
Hello! Thanks for your help, that was a big one 😅. I've requested minor changes and I have asked an open question concerning "étiquette graphql" that we can discuss further.

File: docs/docs/recipes/querying-data.md (hunk 1 above)
Comment (rnd-debug) [reply docs/docs/recipes/querying-data.md]: Fixed. I wasn't confortable with "étiquette" either.

File: docs/docs/recipes/querying-data.md (hunk 2 above)
Comment (rnd-debug) [reply docs/docs/recipes/querying-data.md]: Updated: première mention => "composants NonPageComponent" .Dans la suite, j'utilise uniquement "NonPageComponent" sans le "composant".

File: docs/docs/recipes/querying-data.md (hunk 1 above)
Comment (mindsers) [reply docs/docs/recipes/querying-data.md]: However it seems to be "étiquette" in [other documentations](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Littéraux_gabarits#Gabarits_étiquetés). Maybe we can use the suggestion in https://github.com/gatsbyjs/gatsby-fr/issues/6#issuecomment-716088892 if it helps to understand.

Comment (rnd-debug) [reply docs/docs/recipes/querying-data.md]: Je pense que "étiquette" serait plus "approprié" à un utilisateur [québécois](https://www.btb.termiumplus.gc.ca/publications/media-soc-fra.html), là où l'utilisateur français lambda trouverait ça louche.
J'aime bien la suggestion d'avoir les deux termes en même temps => updated.

**Review by mindsers — APPROVED** [review]
A new PR merged! Thanks again for your help! 🚀 ❤️

**Comment by maa786** [comment]
Perfect translation for my testes. Let’s merge this!

**Pull request merged**